### PR TITLE
Submit copy seed to clipboard with enter

### LINKED ISSUE #3059: Submit copy seed to clipboard with enter
After typing "I understand the risks", copy seed to clipboard only works if "Confirm Seed Copy" button is clicked with the mouse. If the form is submitted by hitting the Enter key, the seed is not copied to clipboard.

### PR DESCRIPTION
**Pull request by amassarwi**
Closes #3059.

### CODE REVIEW

**Comment by amassarwi** [comment]
Test is currently failing, tried changing `user.click(button)` to `fireEvent.submit()` with no luck.

**Comment by amassarwi** [comment]
@bgptr suggested to wrap the button with a `<form>` tag and it seems to do the trick!

File: test/unit/components/views/GetStaredPage/CreateWallet.spec.js [hunk 1]
```diff
@@ -190,7 +190,7 @@ test("test copy seed view", async () => {
   user.type(inputControl, "I understand the risks");
   expect(confirmSeedCopyButton.disabled).toBe(false);
 
-  user.click(confirmSeedCopyButton);
+  fireEvent.submit(confirmSeedCopyButton);
```
Comment (bgptr) [inline test/unit/components/views/GetStaredPage/CreateWallet.spec.js:193]: This change is not necessary, the test pass without it too.

**Review by bgptr — APPROVED** [review]
LGTM Left just a minor inline comment

**Review by alexlyp — APPROVED** [review]

**Pull request merged**